README.md

```diff
@@ -16,28 +16,35 @@ further updates. Apart from its import path, the new package is identical to
 this one &mdash; it contains the same code and releases, so migration usually
 takes just a few minutes and a handful of shell scripts.
 
+## Migration
+
+### 1. Find all instances of `github.com/bufbuild/connect-go` and replace them with `connectrpc.com/connect`
+
 On Linux, or anywhere with GNU `sed`:
 
 ```sh
 find . -name "*.go" -exec sed -i 's|github.com/bufbuild/connect-go|connectrpc.com/connect|g' {} \;
-go get connectrpc.com/connect@latest
-go mod tidy
 ```
 
-On Macs, or anywhere with BSD `sed`:
+On Mac, or anywhere with BSD `sed`:
 
 ```sh
 find . -name "*.go" -exec sed -i '' 's|github.com/bufbuild/connect-go|connectrpc.com/connect|g' {} \;
+```
+
+### 2. Download the latest `connectrpc.com/connect` package
+
+```sh
 go get connectrpc.com/connect@latest
 go mod tidy
 ```
 
-Going forward, you'll need to use the new `protoc-gen-connect-go` to generate
-code:
+### 3. Use the new `protoc-gen-connect-go` plugin to generate code
 
 ```sh
 go install connectrpc.com/connect/cmd/protoc-gen-connect-go@latest
 ```
 
-If you're using remote plugins on the Buf Schema Registry, switch to the
-[connectrpc/go](https://buf.build/connectrpc/go) plugin.
+If you're using remote plugins on the Buf Schema Registry, consider switching 
+to the [connectrpc/go](https://buf.build/connectrpc/go) plugin so you don't have 
+to worry about local installations.
```

Comment:
If the user has already been using the remote plugin `bufbuild/connect-go`, switching to the new plugin is an absolutely necessary step in the migration, not just a consideration. If this step is omitted, the next code generation will insert references to the old github.com/bufbuild/connect-go again, and break the build.

Can we keep the old version of this paragraph?